webrecorder/browsertrix
# Allow users to set additional page time limits

Resolves #636, follows https://github.com/webrecorder/browsertrix-cloud/pull/742



### Changes
- Renames "Page Time Limit" to "Behavior Timeout" and changes to seconds
- Enables users to configure "Page Load Timeout" and "Delay Before Next Page" per API instance defaults
- Differentiates limits per page or per crawl with section headings

### Manual testing

1. Log in and go to New Workflow view
2. Go to Limits tab. Verify new fields are shown under "Limit Per Page"
3. Enter time limits. Verify there is no maximum or requirement enforced
4. Go to Review Config tab. Verify time limits render as expected
5. Save Workflow. Verify time limits render in Workflow Detail as expected

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| New Workflow - Limits | <img width="881" alt="Screen Shot 2023-04-04 at 4 18 10 PM" src="https://user-images.githubusercontent.com/4672952/229943541-831e7afc-1ff6-4988-8417-f927b812f287.png"> |
| New Workflow - Review Config | Without values specified:<br /><img width="232" alt="Screen Shot 2023-04-04 at 4 19 08 PM" src="https://user-images.githubusercontent.com/4672952/229943563-9903f8c0-ca15-44da-9ddd-883b44c1b6bb.png"><br />With values specified:<br /><img width="194" alt="Screen Shot 2023-04-04 at 4 19 51 PM" src="https://user-images.githubusercontent.com/4672952/229943567-305e66ba-c78b-4258-914e-f90d68858fcc.png"> |
